Branch-SDK/build.gradle

```diff
@@ -8,10 +8,13 @@ dependencies {
     compile 'com.android.support:support-annotations:22.2.0'
     compile 'com.crashlytics.sdk.android:answers-shim:0.0.6'
 
-    // This is an optional dependency
-    //Please note that the Branch SDK does not require Firebase to operate. This dependency is listed here so there will not be build errors,
+    // --- optional dependencies -----
+    //Please note that the Branch SDK does not require any of the below optional dependencies to operate. This dependency is listed here so there will not be build errors,
     // but the library is *not* added to your app unless you do so yourself. Please check the code in gradle-mvn-push script to see how this works
+
     compile 'com.google.firebase:firebase-appindexing:10.0.1'
+    compile 'com.android.installreferrer:installreferrer:1.0'
```

Comment:
Please make sure Sahil and you can update the docs appropriately if we're recommending this going forward